src/exec.rs

```diff
@@ -78,6 +78,12 @@ pub struct ProjectRelation {
     expr: Vec<Expr>
 }
 
+pub struct LimitRelation {
+    schema: Schema,
+    input: Box<SimpleRelation>,
+    limit: i64,
```

Comment:
It might be better to use usize here? I don't think we need to support negative limits?